lib/kvstore.sh

```diff
@@ -33,6 +34,17 @@ kv_get() {
   fi
 }
 
+kv_get() {
+  local value=$(kv_get_literal $1 $2 $3)
+
+  # if the value has been wrapped in quotes, unwrap it
+  if [[ $value =~ ^\".+\"$ ]]; then
+    echo "${value:1:${#value}-2}"
```

Comment:
is this a capture group thing?

Reply:
This strips the first and last characters off a string iiuc